sympy/parsing/latex/latex.lark

```diff
@@ -0,0 +1,122 @@
+ADD: "+"
```

Comment:
Is there a page that documents the syntax for this file that we can link to in here (assuming it allows comments)?

Reply:
Here's some links:
- the canonical docs: https://lark-parser.readthedocs.io/en/latest/grammar/
- the simplified self-description: https://github.com/lark-parser/lark/blob/master/examples/lark.lark
- the self-description with templates: https://github.com/lark-parser/lark/blob/master/examples/template_lark.lark
- a python: https://github.com/lark-parser/lark/blob/master/examples/python3.lark

And indeed, comments are denoted with `//`

Reply:
Here is the lark documentation on it https://lark-parser.readthedocs.io/en/latest/grammar/. Though I am sure there are better documents on this.

Reply:
Ah, I thought maybe // were comments, but the lines here confused me. Is there any reason they are commented out?

Reply:
Still working on matching features of the antlr parser